### ISSUE TITLE
Data split of CLS dataset

### ISSUE THREAD

**Issue by CZWin32768** [n-waves/multifit#53]
Could you please provide the data split in your experiments? I couldn't find it in your paper or the [script](https://github.com/n-waves/ulmfit-multilingual/blob/master/prepare_cls.py).
I can only find train, test and unlabeled set. Where is the dev/valid set?
Thanks.

**Comment by PiotrCzapla** [comment]
Hi  the validation set is set to be first 10% of the training set examples:
https://github.com/n-waves/multifit/blob/pretrain-lm/multifit/datasets/dataset.py#L149-L153
